React/Base/RCTTouchHandler.m

```diff
@@ -150,6 +150,11 @@ - (void)_updateReactTouchAtIndex:(NSInteger)touchIndex
   reactTouch[@"locationX"] = @(touchViewLocation.x);
   reactTouch[@"locationY"] = @(touchViewLocation.y);
   reactTouch[@"timestamp"] =  @(nativeTouch.timestamp * 1000); // in ms, for JS
+  
+  if([UIDevice currentDevice].systemVersion.floatValue >= 9) {
+    reactTouch[@"force"] = @(nativeTouch.force);
+    reactTouch[@"maxForce"] =  @(nativeTouch.maximumPossibleForce);
```

Comment:
@kmagiera I agree. Hence force/maxForce (it's going to be "like" one or the other but not being exactly like iOS keeps thing on level ground)